Makefile

```diff
@@ -55,4 +69,103 @@ lint: fmt vet golangci-lint
 test-unit:
 	echo "blah blah blah test"
 
-
+.PHONY: local-env-setup
```

Comment:
do you think we should check to see if the tools are installed here and run the tool make targets if there not like kustomise yq etc. i didnt have them installed in my bin and it failed. Could add it to the make info or help too as a fyi